components/eam/src/dynamics/se/dyn_comp.F90

```diff
@@ -317,7 +324,21 @@ subroutine dyn_init2(dyn_in)
           ! new run, scale mass to value given in namelist, if needed
           call prim_set_mass(elem, TimeLevel,hybrid,hvcoord,nets,nete)
        endif
+       call cldera_dynamic_tracers_pv_idx(pv_idx)
+       if (pv_idx /= 0) then
+          do ie=nets,nete
+            call get_pot_vort(elem(ie), elem(ie)%state%Q(:,:,:,pv_idx), hvcoord, TimeLevel%n0)
+            elem(ie)%state%Q(:,:,:,pv_idx) = elem(ie)%state%Q(:,:,:,pv_idx)+cldera_dynamic_tracers_pv_offset
+          end do
+       end if
 
+       call cldera_dynamic_tracers_pt_idx(pt_idx)
```

Comment:
This block doesn't look like it does anything, is that correct? If so, can we remove it?

Reply:
@OkayHughes is that correct?

Reply:
If you remove the block initializing the PT tracer (line 338), then the PT tracer will be identically zero. I suspect this would not the desired behavior? Apologies if we didn't decide to keep the PT tracer in.

Reply:
I think maybe this comment was meant for a different block, this looks okay to me.